src/WalkingModule/include/WalkingControllers/WalkingModule/Module.h

```diff
@@ -66,6 +70,68 @@ namespace WalkingControllers
+        iDynTree::Twist m_currentFootLeftTwist;/**< The current adapted twist of  the left foot that has been found in previous control cycle. */
+        iDynTree::Twist m_currentFootRightTwist;/**< The current adapted twist of  the right foot that has been found in previous control cycle. */
+        iDynTree::SpatialAcc m_currentFootLeftAcceleration; /**< The current adapted acceleration of  the left foot that has been found in previous control cycle. */
+        iDynTree::SpatialAcc m_currentFootRightAcceleration;/**< The current adapted acceleratiom of  the right foot that has been found in previous control cycle. */
+
+        std::shared_ptr<FootPrint> m_jleftFootprints; /**< The left foot prints */
+        std::shared_ptr<FootPrint> m_jRightFootprints;/**< The right foot prints */
+
+        iDynTree::Vector2 m_zmpToCenterOfFootPositionLeft;
+        iDynTree::Vector2 m_zmpToCenterOfFootPositionRight;
+
+        StepList m_jLeftstepList; /**< The list of left foot steps */
+        StepList m_jRightstepList;/**< The list of right foot steps */
+
+        StepAdapterOutput m_outputStepAdaptation;/**< The structure of outputs of step adaptation */
+        StepAdapterInput m_inputStepAdaptation;/**< The structure of inputs of step adaptation */
+
+        double m_isPushActive;/**< Is push recovery active? */
+        double m_isRollActive;/**< Is the threshold of roll angles of arm active? */
+        double m_isPitchActive;/**< Is the threshold of pitch angles of arm active? */
+        int m_pushRecoveryActiveIndex;/**< The number of control cycles that push recovery is active? */
+        double m_kDCMSmoother;/**< The gain for smoothing of the DCM trajectories */
+        double m_kFootSmoother;/**< The gain for smoothing of the feet trajectories */
+        int m_indexSmoother;/**< The index for the control cycle number of the  DCM trajectories smoothing */
+        int m_indexFootSmoother;/**< The index for the control cycle number of the  feet trajectories smoothing */
+        int m_timeIndexAfterPushDetection;/**< The index for the control cycle number after detecting the push that is used for DCM smoothing*/
+        int m_FootTimeIndexAfterPushDetection;/**< The index for the control cycle number after detecting the push that is used for feet trajectory smoothing*/
+
+        iDynTree::Transform m_smoothedFootLeftTransform;/**< The smoothed transform of left foot after adaptation. */
+        iDynTree::Transform m_smoothedFootRightTransform;/**< The smoothed transform of right foot after adaptation. */
+        iDynTree::Twist m_smoothedFootLeftTwist;/**< The smoothed twist of left foot after adaptation. */
+        iDynTree::Twist m_smoothedFootRightTwist;/**< The smoothed twist of right foot after adaptation. */
+        iDynTree::Vector2  m_DCMPositionSmoothed;/**< The smoothed position of DCM after adaptation. */
```

Comment:
please think to moe this in a separate class

Reply:
Actually as we discussed T2T we will solve it generally after some other modification inside walking module.